tests/test_array/good/fold.catala_en

```diff
@@ -32,7 +32,8 @@ scope B:
 ```catala-test-inline
 $ catala Interpret -s A
 [RESULT] Computation successful! Results:
-[RESULT] x =
+[RESULT]
+x =
```

Comment:
Not sure that we want this newline here between `[RESULT]` and `x`

Reply:
Thing is, since we now ensure that pretty-printing is done consistently, the code below will be aligned with the `x =` ; so there is no clean/easy way to have neither the newline, nor the extra indentation as in the `ERROR` examples below.

Reply:
Hum hum, the downside of doing things correctly :) Then we can leave it as it is in this PR right now.